index.html

```diff
@@ -9,6 +9,13 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="div-frontend">
+    <img src="assets/triangle.svg" class="img-frontend">
+    <h1 class="primary-title">Front-End Tools</h1>
+  </div>
+  <div class="div-frontend">
```

Comment:
Change this class name because you are using it for "frontend" and "backend" sections. And don't use "div" on it. :+1: